docs/projects/ad7134_fmc/index.rst

```diff
@@ -0,0 +1,269 @@
+universal precision data acquisition solutions supporting a breadth of sensor 
+types, from temperature and pressure to vibration and shock.
+
+It supports a wide range of ODR frequencies, from 0.01 kSPS to 1496 kSPS
+with less than 0.01 SPS adjustment resolution, allowing the user to granularly vary
+sampling speed to achieve coherent sampling. The adc offers three main digital 
+filter profile options: a wideband low ripple filter (2.5 kSPS to 374 kSPS),
+a fast responding sinc3 filter (0.01 kSPS to 1496 kSPS), and a balanced sinc6 filter 
+with an ODR range 2.5 kSPS to 1.496 MSPS.
+
+
+The :adi:`AD7134` supports two device configuration schemes: serial peripheral 
+interface (SPI) and hardware pin configuration (pin control mode). The SPI control 
+mode offers access to all the features and configuration options available on the chip.
+Pin control mode offers the benefit of simplifying the device configuration, 
+enabling the device to operate autonomously after power-up operating in a 
+standalone mode. In addition to the optional SPI, it has a flexible and 
+independent data interface for transmitting the ADC output data.
+
+The EVAL-AD7134FMCZ evaluation kit features the AD7134 24-bit, 4-channel, 
+simultaneous sampling adc. Two on-board AD7134 ADCs are included to demonstrate 
+multidevice simultaneous sampling. The HDL reference design provides all the 
+interfaces necessary to interact with the device using a Xilinx FPGA 
+development board.
+
+Supported boards
+-------------------------------------------------------------------------------
+
+-  :adi:`EVAL-AD7134FMCZ <EVAL-AD7134FMCZ>`
+
+Supported devices
+-------------------------------------------------------------------------------
+
+-  :adi:`AD7134`
+
+Supported carriers
+-------------------------------------------------------------------------------
+
+-  :xilinx:`ZedBoard <products/boards-and-kits/1-8dyf-11.html>` on FMC slot *
+
+.. admonition:: Legend
+   :class: note
+
+   ``*`` FMC extender needed for connecting the EVAL board
+
+Block design
+-------------------------------------------------------------------------------
+
+The reference design uses the SPI Engine Framework to interface with the two 
+AD7134 ADCs. It only supports the slave mode for both devices with both DCLK 
+and ODR generated by the FPGA. Each device sends data on 4 of the 8 DIN bits.
+
+Block diagram
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The data path and clock domains are depicted in the below diagrams:
+
+.. image:: ad713x_hdl.svg
+   :width: 800
+   :align: center
+   :alt: AD7134-FMC/ZED block diagram
+
+Jumper setup
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+================== ========= ====================================
+Jumper/Solder link Position  Description
+================== ========= ====================================
+JP14	             Mounted	 DEC0/DCLKIO (Device 1) [DCLK Input]  
```

Comment:
Please align the text in the columns with the table headers and remove the white spaces before "Mounted" and "DEC0...". If needed you can add extra === delimiters to fit the text.

Reply:
This comment is valid for all the following tables.